src/zenml/integrations/gcp/artifact_stores/gcp_artifact_store.py

```diff
@@ -29,22 +29,19 @@
 import gcsfs
 
 from zenml.artifact_stores import BaseArtifactStore
+from zenml.integrations.gcp import GCP_ARTIFACT_STORE_FLAVOR
 from zenml.io.utils import convert_to_str
-from zenml.stack.stack_component_class_registry import (
-    register_stack_component_class,
-)
 
 PathType = Union[bytes, str]
 
 
-@register_stack_component_class
 class GCPArtifactStore(BaseArtifactStore):
     """Artifact Store for Google Cloud Storage based artifacts."""
 
     _filesystem: Optional[gcsfs.GCSFileSystem] = None
 
     # Class Configuration
-    FLAVOR: ClassVar[str] = "gcp"
+    FLAVOR: ClassVar[str] = GCP_ARTIFACT_STORE_FLAVOR
```

Comment:
Good catch! I actually reverted the change from `gcp` to `gcs`. It was the only instance where a stack component flavor didn't have the same name as its integration and I thought it might be a bit unintuitive at this stage.